main/src/ca/uwaterloo/flix/language/phase/MonoTypes.scala

```diff
@@ -1,355 +0,0 @@
-/*
```

Comment:
This is gone for the simply reason that we no longer monomorph types?

Reply:
currently we convert `Option[Int32]` into `Option`. This new approach just leaves `Option[Int32]` as it is. The same goes for structs, they are maintained as they are instead of unfolded into their element types.